src/passport-saml/multiSamlStrategy.ts

```diff
@@ -8,35 +8,31 @@ import {
   AuthorizeOptions,
   MultiSamlConfig,
   RequestWithUser,
+  SamlConfig,
   VerifyWithoutRequest,
   VerifyWithRequest,
 } from "./types";
 
 class MultiSamlStrategy extends SamlStrategy {
-  _options: MultiSamlConfig;
+  _options: SamlConfig & MultiSamlConfig;
 
   constructor(options: MultiSamlConfig, verify: VerifyWithRequest);
   constructor(options: MultiSamlConfig, verify: VerifyWithoutRequest);
   constructor(options: MultiSamlConfig, verify: never) {
-    if (!options || typeof options.getSamlOptions != "function") {
+    if (!options || typeof options.getSamlOptions !== "function") {
       throw new Error("Please provide a getSamlOptions function");
     }
 
-    if (!options.requestIdExpirationPeriodMs) {
-      options.requestIdExpirationPeriodMs = 28800000; // 8 hours
-    }
-
-    if (!options.cacheProvider) {
-      options.cacheProvider = new InMemoryCacheProvider({
-        keyExpirationPeriodMs: options.requestIdExpirationPeriodMs,
-      });
-    }
+    const samlConfig: SamlConfig & MultiSamlConfig = {
+      ...options,
+      cert: options.cert ?? "fake cert", // This would never be a valid cert, to it serves to be a good placeholder
```

Comment:
Suggest change "to" to "so" in comment.
// This would never be a valid cert, **so** it serves to be a good placeholder